gossipd/gossip.c

```diff
@@ -291,6 +291,22 @@ static void reached_peer(struct daemon *daemon, const struct pubkey *id,
 	tal_free(r);
 }
 
+static void queue_peer_msg(struct peer *peer, const u8 *msg TAKES)
+{
+	const u8 *send;
+
+	if (peer->local) {
+		msg_enqueue(&peer->local->peer_out, msg);
+		return;
+	}
```

Comment:
I try to personally avoid tearing apart an if-else statement into if-return, especially when both branches are equally likely. It just helps with readability.

Reply:
Agreed.